.circleci/config.yml

```diff
@@ -272,7 +272,7 @@ jobs:
       - restore_workspace
       - run:
           name: Lint
-          command: make -j8 for-all-target TARGET=lint
+          command: make -j2 for-all-target TARGET=lint
```

Comment:
Perhaps no parallelism at all is the right approach if it causes problems? -j2 appears to take the same time as -j8. Perhaps it does not even benefit from parallelizing.

Reply:
@tigrannajaryan looks like going to -j1 takes ~1 minute longer than -j2 so there is some benefit. I'm not sure if the cache warnings are related or not. I saw one on my local machine but it didn't result in the process killed like I've seen in the CI which I assume was due to resource usage. Maybe let's try -j2 and see if we see any further issues?